src/governor.cairo

```diff
@@ -1,26 +1,49 @@
-use governance::token::ITokenDispatcherTrait;
-use starknet::{ContractAddress};
+use governance::governance_token::IGovernanceTokenDispatcherTrait;
+use starknet::{ContractAddress, StorePacking};
 use array::{Array};
-use governance::token::{ITokenDispatcher};
+use governance::governance_token::{IGovernanceTokenDispatcher};
 use starknet::account::{Call};
+use option::{Option, OptionTrait};
+use integer::{u128_safe_divmod, u128_as_non_zero};
+use traits::{Into, TryInto};
+
+#[derive(Copy, Drop, Serde, PartialEq)]
+struct ProposalTimestamps {
+    // the timestamp when the proposal was created
+    creation: u64,
+    // the timestamp when the proposal was executed
+    executed: u64,
+}
+
+impl ProposalTimestampsStorePacking of StorePacking<ProposalTimestamps, u128> {
+    fn pack(value: ProposalTimestamps) -> u128 {
+        value.creation.into() + (value.executed.into() * 0x10000000000000000_u128)
```

Comment:
would probably be better to have it defined as a constant
`const TWO_POW_64_MINUS_ONE: u128 = 0x10000000000000000;`
for clarity

Also generally speaking you shouldn't need to use the `_u128` suffix I think it should be inferred

Reply:
done, it's 2**64 though, no minus one